Regression in offsets caused offsets to be no longer hashable (#37288)
pandas-dev/pandas: doc/source/whatsnew/v1.1.4.rst

- Fixed regression in :class:`RollingGroupby` causing a segmentation fault with Index of dtype object (:issue:`36727`)
- Fixed regression in :meth:`DataFrame.resample(...).apply(...)` raised ``AttributeError`` when input was a :class:`DataFrame` and only a :class:`Series` was evaluated (:issue:`36951`)
- Fixed regression in :class:`PeriodDtype` comparing both equal and unequal to its string representation (:issue:`37265`)

> - Fixed regression in :meth:`pd.offsets.Day() <pandas.tseries.offsets.Day>` and associated functions caused that offsets were no longer hashable (:issue:`37267`)

Reviewer: ```suggestion
- Fixed regression in certain offsets (:meth:`pd.offsets.Day() <pandas.tseries.offsets.Day>` and below) no longer being hashable (:issue:`37267`)
```

(it's not only Day, but basically all Tick subclasses)

Author: Yeah thats right, should have added that immediately.
